src/pages/debrief/debrief.html

```diff
@@ -19,6 +19,72 @@ <h1>unsuccessful</h1>
 
 <ion-content>
 
+  <ion-card id="dangerous-fault" *ngIf="(pageState.dangerousFaults$ | async).length > 0">
+    <ion-card-header>
+      <ion-card-title>
+        <h1 style="display:inline">{{(pageState.dangerousFaults$ | async).length}}</h1> <h4 style="display:inline">Dangerous</h4>
```

Comment:
Could do with getting rid of these inline styles (all severity levels).

Reply:
added to scss